app/_hub/kong-inc/kafka-log/index.md

```diff
@@ -32,87 +33,103 @@ params:
       value_in_examples: BOOTSTRAP_SERVERS
       urlencode_in_examples: true
       default:
+      datatype:
```

Comment:
@murillopaula I'm not sure about this one. 

Also, is the description correct? Should `{host: host, port: port}` be `{host:port, host: port}`?